web/src/lib/static/loadQuickActionLicenseReinstatements.test.ts

```diff
@@ -0,0 +1,185 @@
+      expect(quickActionTasks).toHaveLength(2);
+      expect(quickActionTasks).toEqual(
+        expect.arrayContaining([
+          {
+            name: "quick action license reinstatement name1",
+            icon: "test.svg",
+            filename: "some filename",
+            urlSlug: "urlslug1",
+            contentMd: "Some content description 1",
+            callToActionLink: "CallToActionLink1",
+            callToActionText: "CallToActionText1",
+            form: "Form1",
+          },
+          {
+            name: "quick action license reinstatement name2",
+            icon: "test2.svg",
+            filename: "some filename 2",
+            urlSlug: "urlslug2",
+            contentMd: "Some content description 2",
+            callToActionLink: "CallToActionLink2",
+            callToActionText: "CallToActionText2",
+            form: "Form2",
+          },
+        ])
+      );
+    });
+  });
+
+  describe("loadQuickActionLicenseReinstatementsByUrlSlug", () => {
+    it("returns quick actions license reinstatement by matching given url slug", () => {
+      const quickActionLicenseReinstatement1 =
+        "---\n" +
+        "filename: some filename 1\n" +
+        "name: quick action license reinstatement name1\n" +
+        "icon: test1.svg\n" +
+        "urlSlug: urlslug1\n" +
+        "callToActionLink: CallToActionLink1\n" +
+        "callToActionText: CallToActionText1\n" +
+        "form: Form1\n" +
+        "---\n" +
+        "Some content description 1";
+
+      const quickActionLicenseReinstatement2 =
+        "---\n" +
+        "filename: some filename 2\n" +
+        "name: quick action license reinstatement name2\n" +
+        "icon: test2.svg\n" +
+        "urlSlug: urlslug2\n" +
+        "callToActionLink: CallToActionLink2\n" +
+        "callToActionText: CallToActionText2\n" +
+        "form: Form2\n" +
+        "---\n" +
+        "Some content description 2";
+
+      mockReadDirReturn({ value: ["opp1.md", "opp2.md"], mockedFs });
+      mockedFs.readFileSync
+        .mockReturnValueOnce(quickActionLicenseReinstatement1) // read first file in list
+        .mockReturnValueOnce(quickActionLicenseReinstatement2) // read second file in list
+        .mockReturnValueOnce(quickActionLicenseReinstatement2); // read file once we found the match
+
+      const quickActionTask = loadQuickActionLicenseReinstatementsByUrlSlug("urlslug2");
+      expect(quickActionTask).toEqual({
+        name: "quick action license reinstatement name2",
+        icon: "test2.svg",
+        filename: "some filename 2",
+        urlSlug: "urlslug2",
+        contentMd: "Some content description 2",
+        callToActionLink: "CallToActionLink2",
+        callToActionText: "CallToActionText2",
+        form: "Form2",
+      });
+    });
+  });
+
+  describe("loadAllQuickActionLicenseReinstatementsUrlSlugs", () => {
+    it("returns quick actions license reinstatement by matching given url slug", () => {
```

Comment:
[pebble] the test says matching url slug but the test is returning all quick action reinstatements?